google-cloud-storage/src/test/java/com/google/cloud/storage/conformance/retry/State.java

```diff
@@ -301,6 +309,30 @@ public boolean hasComposeRequest() {
     return hasValue(KEY_COMPOSE_REQUEST);
   }
 
+  public boolean hasPubsubTopic() {
+    return hasValue(KEY_PUBSUB_TOPIC);
+  }
+
+  public TopicName getTopic() {
```

Comment:
inconsistent names
```suggestion
  public boolean hasPubsubTopic() {
    return hasValue(KEY_PUBSUB_TOPIC);
  }

  public TopicName getPubsubTopic() {
```

nit: `topicName` might make more sense than `pubsubTopic`